src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -92,17 +88,9 @@ class AuthScreens extends React.Component {
     }
 
     componentDidMount() {
-        NetworkConnection.listenForReconnect();
-        PusherConnectionManager.init();
-        Pusher.init({
-            appKey: CONFIG.PUSHER.APP_KEY,
-            cluster: CONFIG.PUSHER.CLUSTER,
-            authEndpoint: `${CONFIG.EXPENSIFY.URL_API_ROOT}api?command=Push_Authenticate`,
-        }).then(() => {
-            Report.subscribeToUserEvents();
-            User.subscribeToUserEvents();
-            Policy.subscribeToPolicyEvents();
-        });
+        Report.subscribeToUserEvents();
```

Comment:
We don't wait for Pusher to initialize – instead `Pusher.subscribe` just internally waits for initialization to complete before setting up the subscription.